x/incentives/keeper/distribute_test.go

```diff
@@ -1198,168 +1200,6 @@ func (s *KeeperTestSuite) WithBaseCaseDifferentInternalGauges(baseCase GroupCrea
 	return baseCase
 }
 
-func (s *KeeperTestSuite) TestCreateGroupAndDistribute() {
```

Comment:
Note: this test will be replaced by https://github.com/osmosis-labs/osmosis/issues/6526